lib/src/ui/full_search_result.dart

```diff
@@ -56,8 +56,12 @@ class FullResultsPage extends StatelessWidget {
                   var websites = getWebsites(result.data['civicrmWebsiteJmaQuery']['entities']);
                   var serviceListingOrg = result.data['civicrmContactById'];
                   var langCode = Localizations.localeOf(context).languageCode.toUpperCase();
-                  var isVerified = (serviceListingOrg["custom911"] != null && serviceListingOrg["custom911"] != 'None' && serviceListingOrg["custom895"] != null);
-
+                  var isVerified = (serviceListingOrg["custom911"] != null
+                      && serviceListingOrg["custom911"] != ''
+                      && serviceListingOrg["custom911"] != ''
```

Comment:
@monishdeb is there a reason why you have duplicated this line?

Reply:
Good catch. Fixed it by removing the extra line